### DISCUSSION TITLE
Publish to server from UDP stream - ingest on v0.21.6_linux_armv7

### DISCUSSION THREAD

**Discussion by peterkelly88** [bluenviron/mediamtx#1603] [Questions] [answered]
The documentation says that it supports UDP ingest. When I follow the instructions and replace all settings in paths like below, the server always responds with ERR: invalid source: 'udp://238.0.0.1:1234'.

Can anyone confirm that it really does support UDP ingest?

paths:
  udp:
    source: udp://238.0.0.1:1234

Thank you.

**Comment by aler9** [comment, marked as answer]
The UDP source feature was added recently and is not yet part of a release. You can either wait for the next release or build the server from source by following instructions in the README.

[a comment by an automated account is not shown]
